_includes/head.html

```diff
@@ -8,6 +9,20 @@
 body {
   font-family: 'Ubuntu';
 }
+
+.mdl-textfield__label:after {
+    background-color: rgb(191, 0 , 0);
```

Comment:
Small suggestion, but to conform with the rest of the files, we should do 2 spaces instead of 4.